lib/ts/recipe/multifactorauth/recipe.ts

```diff
@@ -258,4 +260,52 @@ export default class Recipe extends RecipeModule {
         }
         return result;
     };
+
+    async checkIfLinkingAllowed(session: SessionContainerInterface, factorIds: string[], userContext: UserContext) {
```

Comment:
Because this is supposed to be called to check if MFA allows linking or not. Which maps to an MFA setup, but I think it's more fitting to do that mapping in the MFA recipe instead of the auth utils.